pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -462,29 +470,37 @@ export class SETroubleshootPage extends React.Component {
             />
         );
 
-        var errorMessage;
+        let errorMessage;
         if (this.props.error) {
             errorMessage = (
-                <Alert isInline
-                    variant='danger' title={this.props.error}
-                    actionClose={<AlertActionCloseButton onClose={this.handleDismissError} />} />
+                <AlertGroup isToast>
+                    <Alert
```

Comment:
When using Alerts inside Toast you should use isLiveRegion property which adds the aria-live="polite"  This helps assistive technoligies somehow.

Reply:
Btw opened just now https://github.com/patternfly/patternfly-react/issues/5477

Reply:
thanks, fixed!